test/odbc/psqlodbc/tinyint.cpp

```diff
@@ -0,0 +1,445 @@
+#include "psqlodbc_tests_common.h"
+
+const string TABLE_NAME = "master_dbo.tinyint_table_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+const string DATATYPE_NAME = "sys.tinyint";
+const string VIEW_NAME = "master_dbo.tinyint_view_odbc_test";
+
+const vector<pair<string, string>> TABLE_COLUMNS = {
+    {COL1_NAME, "INT PRIMARY KEY"},
+    {COL2_NAME, DATATYPE_NAME}
+};
+
+const int TINYINT_BYTES_EXPECTED = 1;
+
+class PSQL_DataTypes_TinyInt : public testing::Test {
+
+  void SetUp() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
+      GTEST_SKIP() << "PSQL Driver not present: skipping all tests for this fixture.";
+    }
+
+    OdbcHandler test_setup(Drivers::GetDriver(ServerType::PSQL));
+    test_setup.ConnectAndExecQuery(DropObjectStatement("TABLE", TABLE_NAME));
+  }
+
+  void TearDown() override {
+    if (!Drivers::DriverExists(ServerType::PSQL)) {
+      GTEST_SKIP() << "PSQL Driver not present: skipping all tests for this fixture.";
+    }
+
+    OdbcHandler test_teardown(Drivers::GetDriver(ServerType::PSQL));
+    test_teardown.ConnectAndExecQuery(DropObjectStatement("VIEW", VIEW_NAME));
+    test_teardown.CloseStmt();
+    test_teardown.ExecQuery(DropObjectStatement("TABLE", TABLE_NAME));
+  }
+};
+
+// Helper function to convert string to equivalent C version of int.
+int StringToTinyInt(const string &value) {
```

Comment:
This is the same as `StringToInt4` in `int.cpp`. Can we move this to the psql_commons? There is also a similar function in bigint if I'm not mistaken (but uses strtol instead of stoi), may be a good idea as well to join that in there.

Reply:
Moved to `conversion_functions_common`. Created a new commons file so the other one won't be cluttered.